### ISSUE TITLE
mouse-over/tooltip texts

### ISSUE THREAD

**Issue by arlogriffiths** [erc-dharma/project-documentation#291]
@danbalogh and @michaelnmmeyer —

I notice that several of the little descriptions that pop up with mouse-over (I Dan calls this tooltip) are neither direct expressions of the TEI elements nor identical to what we say on https://dharmalekha.info/editorial-conventions. 

I attach some screenshots as examples:
- why "Line start" and not "Line beginning" for `<lb>`?
- why give more than "Superfluous text" for `<surplus>`?
- why "Foliation work" and not "Forme work" for `<fw>`? (Though since "forme work" is such a specialized term, that I'd never heard before the start of DHARMA in 2019, maybe a more generally understandable expression like "Foliation" without "work" would be desirable.)

Could you work together to make sure all relevant messages have Dan's approval? I am happy to trust his judgment.

![Air Kali (849-850 Śaka) - DHARMA 2](https://github.com/erc-dharma/project-documentation/assets/835603/687ca51b-d278-49e0-b21e-a5b14d1fafa3)
![Air Kali (849-850 Śaka) - DHARMA 3](https://github.com/erc-dharma/project-documentation/assets/835603/0a3c1f2c-5f28-4a6b-b031-29e07791cb28)
![Air Kali (849-850 Śaka) - DHARMA](https://github.com/erc-dharma/project-documentation/assets/835603/8afb4c4d-e4e3-4792-a64a-b90b674444df)

**Comment by danbalogh** [comment]
If @michaelnmmeyer could prepare a list of what tooltip is shown for what code, I should be able to suggest improvements. This could perhaps be integrated with our collaboration on the cheatsheet/conventions list, though it may be better to keep the two separate. I do, however, think that tooltips should be kept relatively simple, and that means that they may not be 100% appropriate in all cases. Thus, @arlogriffiths , I'm not sure what your problem is with the tooltip "Superfluous text erroneously added by the scribe", but if it is that the suppressed comma is not really "text", then I think this is within tolerance.

**Comment by arlogriffiths** [comment]
I meant that "Superfluous text erroneously added by the scribe" seems redundant to me. Why would "Superfluous text" not be sufficient?

**Comment by michaelnmmeyer** [comment]
I will create a sample file that enumerates all possible cases. This will take some time, though.

**Comment by danbalogh** [comment]
Thanks, I don't think this is urgent. If we leave this thread open, I'll get an alert when you mention it here; or you can just send me the sample file directly over email.

**Comment by arlogriffiths** [comment]
Here's another thing that doesn't seem right.

Code:

```
 <lg xml:lang="san-Latn" met="śārdūlavikrīḍita" n="34">
 <l n="a"><supplied reason="lost" evidence="previouseditor">sādhūnām pathi yātu paura-samitir dharmyā gatir ma</supplied>ntriṇām</l>
 <l n="b">bhūyād bhūta-hiteṣiṇo muni-janā Itthan na me prārthanā</l>
<l n="c">yasmiñ jīvati rājñi ra<lb break="no" n="A36"/><supplied reason="lost" evidence="previouseditor">kṣati bhuvan dharmeṇa siddhyanti te</supplied></l>
<l n="d"><supplied reason="lost" evidence="previouseditor">tasmāc chrī-jala</supplied>laṅga-deva-nr̥-patir dīrghaṁ sa jīvyād iti ⊙ // ⊙</l>
</lg>
```

Display: 
<img width="472" alt="Capture d’écran 2024-04-23 à 20 05 00" src="https://github.com/erc-dharma/project-documentation/assets/835603/68935c24-ac0d-48cd-b277-32e838e7c0b7">


Note wrong explanation of @evidence of `<supplied>`.

**Comment by arlogriffiths** [comment]
Can we get this moving again? What about starting with the correction of the wrong tooltip "Foliation work"? Above I suggested that "Foliation" would be good.

<img width="337" alt="Capture d’écran 2024-06-28 à 07 04 31" src="https://github.com/erc-dharma/project-documentation/assets/835603/308f065d-424f-42d3-9251-f3f864ea3b75">

**Comment by michaelnmmeyer** [comment]
OK for "Foliation".

**Comment by danbalogh** [comment]
Do we still have the time and energy for this? I.e. for @michaelnmmeyer to create a list of the tooltips used in the display and for @arlogriffiths and me to check it over and suggest improvements.

**Comment by arlogriffiths** [comment]
I think this is important and don't understand why it didn't get done in 2024. As a consequence, it simply has to get done now.

**Comment by michaelnmmeyer** [comment]
I cannot give a you a list of tooltips because they are generated algorithmically. There are hundreds of variants, and the result depends on input parameters. Basically you are asking to rewrite the generation system in pseudocode that you would be able to decipher. This would take too much time at this stage. Note that I have only 1 week and a half left to work on DHARMA.

**Comment by arlogriffiths** [comment]
@michaelnmmeyer:
- does it mean that you could not even change the cases that I have flagged, with the alternatives I propose, if you wanted to?
- I don't understand how/where an algorithm intervenes to shows "Line start" instead of "Line beginning" for `<lb/>` and would like to receive a basic explanation

**Comment by michaelnmmeyer** [comment]
@arlogriffiths 

All the cases you flagged have already been addressed, except `MILESTONE start`  > `MILESTONE beginning`. I put MILESTONE in capitals because it is a placeholder that is replaced like this:

* If the element is `<lb>`, `MILESTONE` is set to `Line`
* If the element is `<milestone>` and a `@unit` is present,  `MILESTONE` is set to the unit name, with an initial capital. You thus end up with `Page start`, `Face start`, `Fragment start`, etc., depending on `@unit`.
* If the element is `<milestone>` and no `@unit` is present, `MILESTONE` is set to `Column`.

Basically, most tooltips are parameterized. Their value depends on the element itself, on its attributes or its contents, even sometimes on the enclosing elements.
